x-pack/filebeat/module/threatintel/abusemalware/manifest.yml

```diff
@@ -0,0 +1,16 @@
+module_version: 1.0
+
+var:
+  - name: input
+    default: httpjson
+  - name: interval
+    default: 60m
+  - name: url
+    default: "https://urlhaus-api.abuse.ch/v1/payloads/recent/"
+  - name: ssl
```

Comment:
Please ensure parameters without defaults are protected with `{{ if .X }}....{{ end }}` in the templates. I'm pretty sure our renderer is setup to error on undefined variables.